src/lib/enums/tuning-resources.ts

```diff
@@ -74,17 +76,19 @@ enum TuningResourceType {
   SimFilter = 0x6E0DDA9F,
   SimInfoFixup = 0xE2581892,
   SimTemplate = 0x0CA4C78B,
-  SituationGoalSet = 0x9DF2F1F2,
+  Situation = 0xFBC3AEEB,
   SituationGoal = 0x598F28E7,
+  SituationGoalSet = 0x9DF2F1F2,
   SituationJob = 0x9C07855F,
-  Situation = 0xFBC3AEEB,
-  SlotTypeSet = 0x3F163505,
   SlotType = 0x69A5DAA4,
+  SlotTypeSet = 0x3F163505,
   Snippet = 0x7DF2169C,
   SocialGroup = 0x2E47A104,
   Spell = 0x1F3413D9,
-  Scommodity = 0x51077643,
+  StaticCommodity = 0x51077643,
```

Comment:
Probably would have been best to use the full name from the start, but changing this now is a breaking change that I would like to avoid.

Reply:
sadge but ok